src/server/server.rs

```diff
@@ -566,8 +566,10 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Handler for Server<T, S> {
         // nothing to do now.
     }
 
-    fn interrupted(&mut self, event_loop: &mut EventLoop<Self>) {
-        event_loop.shutdown();
+    fn interrupted(&mut self, _: &mut EventLoop<Self>) {
+        // To be able to be attached by gdb, we should not shutdown.
```

Comment:
most of case, we won't use gdb for tikv, so I think catching `ctrl + c` and then closing is more convenient. 

Does `handle SIGINT stop` in gdb not work?